### PR TITLE
remove duplicate app: label in selectorLabels helper

### PR DESCRIPTION
**Pull request by aymeric-ledizes**

### CODE REVIEW

File: templates/_helpers.tpl [hunk 1]
```diff
@@ -528,7 +528,6 @@ Selector labels
 {{- define "redash.selectorLabels" -}}
 app.kubernetes.io/name: {{ include "redash.name" . }}
 app.kubernetes.io/instance: {{ .Release.Name }}
-app: {{ .Release.Name }}
```
Comment (frivoire) [inline templates/_helpers.tpl:531]: I understand you're removing a duplicate label.
Why removing it from here and not from the deployment.yaml file for example ?

I guess we should try to keep our patches (fork vs upstream) to a minimum to limit conflicts when rebasing/merging in the future.
So, we should remove the label:
- where _we_ have added it => it seems we have added the 2 duplicates, so any one is ok to remove
- and where it was adding _more lines_ => our patches added it in several files like `adhocworker-deployment.yaml`, `scheduledworker-deployment.yaml`, `server-deployment.yaml`, and maybe other that I haven't seen

So, I have the feeling we should keep it in the helper (BTW: we also have the component:database here, so already a patch) and remove it from the various templates files (cf above)

Comment (aymeric-ledizes) [reply templates/_helpers.tpl]: Yes I simplified it the other way around by keeping the `app` label only in the helper:`redash.selectorLabels` and remove it anywhere else as `redash.labels` helper calls `redash.selectorLabels` (https://github.com/blablacar/redash-contrib-helm-chart/blob/redash-bbcfork-v0.2.4/templates/_helpers.tpl#L516-L518)

Anyway this fork will need a little bit of rework to match the "new way" we manage upstream fetch/rebase like the rabbitmq chart

**Review by frivoire — APPROVED** [review]
Looks nice 👍

File: templates/_helpers.tpl (hunk 1 above)
Comment (frivoire) [reply templates/_helpers.tpl]: Yep, we can improve the forking method here.
My remark was really not to start working on that rework now, but just to go in the right direction (less fork to maintain) when fixing the duplicates (which is today's topic).

**Pull request merged**